subset/pentests/nmap/test_nmap

```diff
@@ -140,7 +140,17 @@ grep -oh '[0-9]*/open[^[:space:]]*' $LOG_HTTP | while IFS=/ read -ra parts; do
     fi
 done
 
-if [ -f $TEST_ROOT/.fail ]; then
+if [[ -f $CONFIG ]]; then
```

Comment:
But it looks like nmap is still running twice regardless of whether the config is there.